compiler/pipes/check-func-calls-and-vararg.cpp

```diff
@@ -281,10 +281,11 @@ VertexPtr CheckFuncCallsAndVarargPass::on_func_call(VertexAdaptor<op_func_call>
       if (FunctionPtr f_callback = call_arg.as<op_callback_of_builtin>()->func_id) {
         int call_n_params = func_param->type_hint->try_as<TypeHintCallable>()->arg_types.size() + call_arg.as<op_callback_of_builtin>()->args().size();
         int delta_this = f_callback->has_implicit_this_arg() ? 1 : 0;
+        auto expected_callback_signature = func_param->type_hint->try_as<TypeHintCallable>()->as_human_readable();
```

Comment:
I would put it in a separate variable, because the same thing is done above:
```c++
const auto *typed_callable = func_param->type_hint->try_as<TypeHintCallable>();
```